phantom/forgejo/Dockerfile

```
@@ -0,0 +1,18 @@
+FROM ubuntu
```

Hadolint (2.14.0)

[warning] Always tag the version of an image explicitly

(DL3006)